test/acceptance-tests/use-cases/test-scripts/create-models.js

```diff
@@ -86,4 +88,49 @@ module.exports = function() {
     expect(testsuite.expectedProperty).to.eql(property._content);
     next();
   });
+
+  this.Given(/^I add relation '(.+)' from '(.+)' to '(.+)'$/,
+  function(relationName, fromModelName, toModelName, next) {
+    testsuite.fromModelName = fromModelName;
+    testsuite.toModelName = toModelName;
+    testsuite.relationName = relationName;
+    next();
+  });
+
+  this.When(/^the relation is of type '(.+)' and foreignKey '(.+)'$/,
+    function(relationType, foreignKey, next) {
+      const relationDef = {
+        id: testsuite.relationName,
+        type: relationType,
+        foreignKey: foreignKey,
+        modelId: testsuite.fromModelName,
+        model: testsuite.toModelName,
+        facetName: 'common',
+      };
+      testsuite.relationTest = {};
+      testsuite.relationTest.facetName = relationDef.facetName;
+      testsuite.relationTest.fromModelName = relationDef.modelId;
```

Comment:
Should the above 3 be moved into the callback? Ie. only create these relationTest* variables if the ModelRelation.create succeeds?